ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -15,6 +15,8 @@ images:
 - dockerfile_path: dev/Dockerfile
   from: base
   to: cincinnati-graph-data-container
+operator:
+  bundles:
```

Comment:
openshift/cincinnati-operator#76 is failing because we don't have a v3 `kustomize` in the test-step image.  But you could be compatible with the operator SDK layout with:

```yaml
operators:
  bundles:
  - dockerfile_path: bundle.Dockerfile
```

here.